### PR TITLE
Add emerald

### PR DESCRIPTION
**Pull request by tgurbich**
- Added a SanntiS annotation step.
- Modified the GFF annotation step to include SanntiS.

### CODE REVIEW

File: cluster/codon/execute/test/test/scripts/step1.test.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (mberacochea) [inline cluster/codon/execute/test/test/scripts/step1.test.sh:1]: Argh... we need to remove this. It's my fault, I accidentaly pushed at some point

File: docker/genomes-catalog-update/scripts/annot_gff.py [hunk 2]
```diff
@@ -46,6 +46,51 @@ def get_eggnog(eggnot_annot):
...
+def get_sanntis(sanntis_file, prokka_gff):
+    cluster_positions = dict()
+    sanntis_result = dict()
+    bgc_annotations = dict()
+    # save positions of each BGC cluster annotated by SanntiS to dictionary cluster_positions
+    # and save the annotations to dictionary sanntis_result
+    with open(sanntis_file, "r") as sanntis_in:
+        for line in sanntis_in:
+            if not line.startswith("#"):
+                cols = line.strip().split("\t")
```
Comment (mberacochea) [inline docker/genomes-catalog-update/scripts/annot_gff.py:58]: A more readable way of splitting this:
```python
contig, _, _ , start_pos, end_pos, _, _, annotations = line.strip().split("\t")
```
that way you can "spread" the values, and assign them directly to variables (the "_" are just unused variables)

**Review by mberacochea — APPROVED** [review]
The changes look OK.

Can you upload an example of an input and the expected output , I'll use them to write an unit test (later)?.

File: docker/genomes-catalog-update/scripts/annot_gff.py (hunk 2 above)
Comment (tgurbich) [reply docker/genomes-catalog-update/scripts/annot_gff.py]: great suggestion, thank you! added the change

File: cwl/tools/emerald/emerald.cwl [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: CommandLineTool
+
+label: "emeraldbgc"
```
Comment (KeteSakharova) [inline cwl/tools/emerald/emerald.cwl:5]: I guess emerald is an old name of tool. Maybe it makes sense to rename a folder and cwl file to sanntis?

Comment (mberacochea) [reply cwl/tools/emerald/emerald.cwl:5]: I'll rename in another branch

**Pull request merged**